engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -34,6 +38,8 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
   val jobTag = s"${workflowId.shortString}:${jobKey.call.fullyQualifiedName}:${jobKey.index.fromIndex}:${jobKey.attempt}"
   val tag = s"EJEA_$jobTag"
 
+  private var callCachingActorProps: Props = _
```

Comment:
I don't like `null`s either, hence the sneaky `var` / `_` 😛 

I could go either way on this and I'm open to doing whatever others think best.  The code that deals with the call cache copying actor only runs if call cache reading is turned on, which it won't be if there isn't a call cache copying actor available.  i.e. no code should try to read this variable if this variable isn't set, so wrapping the variable in an `Option` would mask the violation of that invariant.  On the other hand, `null`s are icky and this is basically a sugared `null`.